src/archlinux/initramfs-suspend

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/ash
+#!/usr/bin/bash
```

Comment:
This isn't a typo. Arch really uses [ash](https://en.wikipedia.org/wiki/Almquist_shell) shell in initramfs scripts.

Reply:
Thanks for the clarification. Fixed in recent commit.